src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.util;
+
+import mate.academy.exception.DataProcessingException;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    public HibernateUtil() {
```

Comment:
From Checklist "Add a private default constructor to HibernateUtil class in order to prevent HibernateUtil’s objects creating"